Classes/CampaignsFactory.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
```

Comment:
strict_types please

Reply:
If that's required, we should add that to CI. I'm not able to know all rules of all OpenSource and Customer Projects I'm contributing to. If something is not checked and enforced, it doesn't count for me. That's what computers are made for. People should not waste time checking and tackling such boring stuff.

As it seems you are forcing that rule, I'll add it to the CGL check and fixer. That will be a separate PR. Once done, everyone is able to rebase this PR and apply the fixer.

Reply:
- [ ] I've already rebased and applied. Please merge https://github.com/SkillDisplay/TYPO3ContentElements/pull/7 before.

Reply:
> As it seems you are forcing that rule, even if it was not applied to existing code, I'll add it to the CGL check and fixer. 

Basically: Any new PHP code not using strict types (or any method to make code more meaningful) is basically crap by definition.
I do not enforce it on existing code, since it was written before I was active here.
So, thanks for getting it straight with the other PR!